packages/fuel-indexer-lib/src/config/mod.rs

```diff
@@ -178,6 +187,22 @@ pub struct ApiServerArgs {
     /// Postgres port.
     #[clap(long, help = "Postgres port.")]
     pub postgres_port: Option<String>,
+
+    /// Run database migrations before starting service.
+    #[clap(
+        long,
+        default_value = "false",
+        help = "Run database migrations before starting service."
+    )]
+    pub run_migrations: bool,
+
+    /// Use Prometheus metrics reporting.
```

Comment:
@deekerno Can we leave a note or something here that this doesn't actually do anything at the moment because we don't track metrics for the api server (yet)?

Reply:
Will do!